botwfstools/botw_patcher.py

```diff
@@ -15,10 +15,19 @@
 import sys
 import typing
 import wszst_yaz0
+import re
 
 ARCHIVE_EXTS = {'sarc', 'pack', 'bactorpack', 'bmodelsh', 'beventpack', 'stera', 'stats',
                 'ssarc', 'spack', 'sbactorpack', 'sbmodelsh', 'sbeventpack', 'sstera', 'sstats',
                 'blarc', 'sblarc', 'genvb', 'sgenvb'}
+AOC_PREFIX = 'Aoc/0010/'
+AOC_PREFIX_CONDITIONS = {'Terrain\/A\/AocField.*', 'Pack\/.*.pack', 'UI\/StaffRollDLC\/.*',
```

Comment:
This should be a tuple or a list; it should not be a set.

Also, the list of conditions is technically not entirely correct. This will match all dungeon resources and all voice streams for example. The dungeon number needs to be checked like how Nintendo does it. Here is a [fairly accurate](https://zeldamods.org/wiki/Module:AocUtils) implementation.

(Using regexes for that many strings is also going to be pretty expensive when you have many resources to patch. Best to use simple string comparisons for most cases and save regex/more complex parsing for the special cases)